aggregate-backup.rst

```diff
@@ -0,0 +1,202 @@
+Backup Strategies for Aggregate
+================================
+
+.. _briefcase-backup:
+
+Recovering from Briefcase
+---------------------------
+
+Use :doc:`ODK Briefcase <briefcase-guide>` to back up all forms and submissions on Aggregate. You can :ref:`pull forms from your Aggregate server <pull-from-aggregate>` into your local machine using Briefcase. In particular, Briefcase's :ref:`command line interface <cli-use>` makes this easier.
+
+.. _mysql-backup:
+
+Recovering database from MySQL dump 
+--------------------------------------
+
+1. Stop running the Tomcat.
+2. Upgrade to the latest version og Aggregate.
+3. Finally restore it from MySQL dump. An SQL dump of a database is a common method to safely store away a snapshot of the database for archival purposes or to migrate data between database instances, e.g. between two major system releases. The content of a SQL dump is a large collection of SQL commands in ASCII. Running the script will recreate the database in the same state as it was when the dump was created. The primary tool to consider for making an ASCII dump is msqldump, which includes a wide variety of options:
+
+.. code-block:: console
+
+	$ msqldump [ options ] [ dbname ]
+
+Options are:
```

Comment:
i don't think we need msqldump docs recreated here. Mention the commands/options our folks are going to use and then link to the msqldump documentation